File: system-test/java/src/test/java/com/optivem/eshop/systemtest/legacy/mod11/contract/clock/BaseClockContractTest.java
package com.optivem.eshop.systemtest.legacy.mod11.contract.clock;

import com.optivem.eshop.systemtest.legacy.mod11.contract.base.BaseExternalSystemContractTest;
import org.junit.jupiter.api.Test;

public abstract class BaseClockContractTest extends BaseExternalSystemContractTest {
    @Test
    void shouldBeAbleToGetTime() {
        scenario
                .given().clock().withTime("2024-01-02T09:00:00Z")
                .then().clock().hasTime();
    }
}

File: system-test/dotnet/SystemTests/Legacy/Mod11/ExternalSystemContractTests/Clock/BaseClockContractTest.cs
using SystemTests.Legacy.Mod11.ExternalSystemContractTests.Base;

namespace SystemTests.Legacy.Mod11.ExternalSystemContractTests.Clock;

public abstract class BaseClockContractTest : BaseExternalSystemContractTest
{
    [Fact]
    public async Task ShouldBeAbleToGetTime()
    {
        (await Scenario()
            .Given()
            .Then().Clock())
            .HasTime();
    }
}










